tests/testthat/test-ggMarginal.R

```diff
@@ -0,0 +1,25 @@
+# Wrap up the ggMarginal visual tests in a function runMarginalTests so that it's easy to test under multiple versions of ggplot2
+runMarginalTests <- function(ggplot2Version) {
+
+  context(ggplot2Version)
+  
+  test_that("ggMarginal can produce basic marginal plots" , {
```

Comment:
I was thinking that - given the number of test cases we are likely to have (maybe twice as many as we have now, total) in the future, it makes more sense to enumerate them like this. They group into the three test that statements in a way that I think works - basic usage, parameters work, then issues are solved.